### ISSUE TITLE
Training vs. Validation loss

### ISSUE THREAD

**Issue by srajamani77** [vqdang/hover_net#167]
In the stat.json as well as output per epoch during training, the losses used for validation are slightly different than that of training. Can you please let me know why they are not the same for training and validation? How can I change the code so that they are the same? Below is my training output from the 1st epoch.
![image](https://user-images.githubusercontent.com/60722207/134237595-e44d106c-c453-4621-822a-9decb83004ce.png)

**Comment by simongraham** [comment]
Here, you are not seeing the validation loss, but you are seeing some calculated metrics that we wish to use to help determine which is the best checkpoint. After all, we want to select the best model based on the segmentation performance, rather than the best HoVer map MSE (for example).

**Comment by srajamani77** [comment]
That makes sense. Thank you for your input.
